playbooks/scenario2c_coreos_az.yaml

```diff
@@ -0,0 +1,111 @@
+---
+# Scenario to test ECS service with CoreOS image in different AZs
+# A VPC is created and VM is being started in different AZs
+#
+
+- name: Scenario 2c - Compute per AZ
+  hosts: localhost
+  vars:
+    prefix: scenario2c-
+    test_image_url: https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/33.20210117.3.2/x86_64/fedora-coreos-33.20210117.3.2-openstack.x86_64.qcow2.xz
+    test_image_name: fedora-coreos-33
```

Comment:
can be gathered from:

```
curl -sS https://builds.coreos.fedoraproject.org/streams/stable.json | jq -r .architectures.x86_64.artifacts.openstack.release
33.20210117.3.2
```

Reply:
as mentioned above - too dangerous from my pov